### PR TITLE
Add loaders for fonts

### PR DESCRIPTION
**Pull request by LasTanzen**
Add loaders for several font types. Don´t know if SVG is really needed, hat no conflicts while using some in templates. All credits go to Adrian Hall (https://shellmonger.com/2016/01/22/working-with-fonts-with-webpack/).

### CODE REVIEW

**Comment by filipesilva** [comment]
@TheLarkInn can you review?

**Comment by TheLarkInn** [comment]
This doesn't look up to date with master can you double check @LasTanzen

**Comment by LasTanzen** [comment]
Thanks for checking @TheLarkInn . Seems that 3b226be already added this. I checked the master branch in IE9 (Windows7), IE11 (Windows8), Edge (Windows10), Chrome, Firefox, Safari (Mac) and had problems with SVG-fonts and Safari due to the mandatory id after the hash in the URL. I fixed this and updated the PR after checking in all browsers again.

**Comment by filipesilva** [comment]
@LasTanzen sorry it took so long to get to this PR. Thanks for the research, it looks good and ready to merge pending a rebase. Can you solve the conflict? I think we added a new font and that's all.

**Comment by LasTanzen** [comment]
Hi @filipesilva, I merged from master and did some browser testing. Cheers

File: addon/ng2/models/webpack-build-common.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
+import * as path from 'path';
```
Comment (filipesilva) [inline addon/ng2/models/webpack-build-common.ts]: This file seems a remnant of a rebase, can you remove it? It's making the CI break.

**Comment by LasTanzen** [comment]
Argh, haven´t realized you meant the whole file. Removed it, sorry for the inconveniences. Hope it works now :)
.

**Comment by filipesilva** [comment]
Thanks, we'll merge it soon!

**Pull request closed without merging**